ci/cpu/prebuild.sh

```diff
@@ -7,21 +7,21 @@ set -e
 export UPLOAD_CUDF=1
 
 #Upload libcudf once per CUDA
-if [[ "$PYTHON" == "3.7" ]]; then
+if [[ "$PYTHON_VER" == "3.7" ]]; then
     export UPLOAD_LIBCUDF=1
 else
     export UPLOAD_LIBCUDF=0
 fi
 
 # upload cudf_kafka for all versions of Python
-if [[ "$CUDA" == "11.0" ]]; then
+if [[ "$CUDA_VER" == "11.0" ]]; then
     export UPLOAD_CUDF_KAFKA=1
 else
     export UPLOAD_CUDF_KAFKA=0
 fi
 
 #We only want to upload libcudf_kafka once per python/CUDA combo
-if [[ "$PYTHON" == "3.7" ]] && [[ "$CUDA" == "11.0" ]]; then
+if [[ "$PYTHON_VER" == "3.7" ]] && [[ "$CUDA_VER" == "11.0" ]]; then
```

Comment:
Are these 2 environment variables `$PYTHON_VER`, `$CUDA_VER` specific to libcudf?
Commonly used name for cuda version is `$CUDA_VERSION`. So, is this naming  done intentionally to avoid collision with existing variables?

Reply:
@karthikeyann, the `$CUDA_VER` environment variable is something that the Ops teams sets explicitly in our CI environments. We use `$CUDA_VER` (previously just `$CUDA`) to differentiate from the `$CUDA_VERSION` environment variable that is set in the `nvidia/cuda` Docker images from which we base our RAPIDS images.